### PR TITLE
Added beautified component names

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Add "beautified" component names
- Change the component name macro
- Fix the R3F compat component name (was BWA3_... before)
- Make all component names lowercase only

### CODE REVIEW

File: addons/laserpointer/script_component.hpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #define COMPONENT laserpointer
+#define COMPONENT_BEAUTIFIED Laserpointer
```
Comment (bux578) [inline addons/laserpointer/script_component.hpp]: Laser pointer

File: addons/logistics_wirecutter/script_component.hpp [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #define COMPONENT logistics_wirecutter
+#define COMPONENT_BEAUTIFIED Logistics Wirecutter
```
Comment (bux578) [inline addons/logistics_wirecutter/script_component.hpp]: wire cutter

File: addons/nametags/script_component.hpp [hunk 3]
```diff
@@ -1,4 +1,5 @@
 #define COMPONENT nametags
+#define COMPONENT_BEAUTIFIED Nametags
```
Comment (bux578) [inline addons/nametags/script_component.hpp]: name tags

**Comment by commy2** [comment]
First time using githubs online commit function
@BaerMitUmlaut I know it's hard for a German. One day _they_ will learn the beauty of "zusammengesetzte Substantive".

**Pull request merged**